lib/plottr_components/src/components/outline/BeatView.js

```diff
@@ -128,8 +176,14 @@ const BeatViewConnector = (connector) => {
           topOffset={'60%'}
           bottomOffset={'60%'}
         >
-          <div>
+          <div
+            onDragLeave={this.handleDragLeave}
+            onDragEnter={this.handleDragEnter}
+            onDragOver={this.handleDragOver}
+            onDrop={this.handleDrop}
+          >
             <h3 id={`beat-${beat.id}`} className={klasses}>
+              {this.renderDropZone()}
```

Comment:
I think that if you put this after the `beatTitle` then it should render beneath the title when you drag and drop.

Reply:
Yeah it looks more consistent. Thank you @Quiescent